cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go

```diff
@@ -53,6 +55,19 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 				EtcdVersion:        "v0.1.0",
 				OperatorVersion:    "v0.1.0",
 			}
+			obj.KubeletConfiguration = kubeadm.KubeletConfiguration{
```

Comment:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/55803/pull-kubernetes-bazel-test/17692/
pull-kubernetes-bazel-test failed here but the failure info is too unreadable to debug...

@luxas @sttts I wonder what a fuzzer does. Should I list all members of `KubeletConfiguration` here and assign them a value? 
:scream: :scream: :scream:

Reply:
The fuzzer fills in random data into types, testing conversions and deepcopy and serialization. Usually, just compare the in and output and trace the code why they are different. Often the conversion really has a bug or the fuzzer needs to be customized to avoid invalid data. Rule of thumb: if the fuzzer is not wrong, those issues are critical as they might lead to data loss and are very hard to debug.

Reply:
@sttts thanks for clarification! I still can't understand the mechanism completely though :) So what should I do to fix the test failure here?

Reply:
I think you need to add the line `obj.FeatureGates = map[string]bool{}` in `NodeConfiguration`